docs/setup.asciidoc

```diff
@@ -147,6 +152,34 @@ NOTE: Be careful not to execute `DbInterception.Add` for the same interceptor mo
 or you'll get additional interceptor instances.
 For example, if you add `Ef6Interceptor` interceptor twice, you'll see two spans for every SQL query. 
 
+[float]
+[[setup-sqlclient]]
+==== SqlClient
+
+You can enable auto instrumentation for `System.Data.SqlClient` or `Microsoft.Data.SqlClient` by referencing `Elastic.Apm.SqlClient` package
+and passing `SqlClientDiagnosticSubscriber` to the `UseElasticApm` method in case of ASP.NET Core as it shown in example:
+
+[source,csharp]
+----
+app.UseElasticApm(Configuration,
+	new SqlClientDiagnosticSubscriber());  /* Enable tracing of outgoing db requests */
+----
+
+or passing `SqlClientDiagnosticSubscriber` to the `Subscribe` method in Global.asax in case of `ASP.NET` as it shown in example:
+
+[source,csharp]
+----
+public override void Init()
+{
+    // you need add custom code to be sure that Subscribe called only once and in a thread-safe manner
+    if (Agent.IsConfigured) Agent.Subscribe(new SqlClientDiagnosticSubscriber());  /* Enable tracing of outgoing db requests */
+}
+
+----
```

Comment:
```suggestion
or passing `SqlClientDiagnosticSubscriber` to the `Subscribe` method and make sure that the code is called only once, otherwise the same database call could be captured multiple times:

[source,csharp]
----
// you need add custom code to be sure that Subscribe called only once and in a thread-safe manner
if (Agent.IsConfigured) Agent.Subscribe(new SqlClientDiagnosticSubscriber());  /* Enable tracing of outgoing db requests */
----